install/debian/InstallHalyard.sh

```diff
@@ -152,6 +192,24 @@ usage: $0 [-y] [--version=<version>] [--user=<user>]
     -y                              Accept all default options during install
                                     (non-interactive mode).
 
+    --halyard-bucket <name>         The bucket the Halyard JAR to be installed
```

Comment:
how do the jars get into this bucket?

Reply:
Talked offline: https://github.com/spinnaker/halyard/blob/master/release/publish.sh